### PR TITLE
fix vsphere connected profile

### PR DESCRIPTION
**Pull request by sgaoshang**
Change segment from devqe-segment-223 to devqe-segment-228

### CODE REVIEW

**Comment by sgaoshang** [comment]
@jianlinliu pls help to review this PR, thanks.

**Comment by sgaoshang** [comment]
@smg247 Could you help to approve it, thanks.

**Comment by sgaoshang** [comment]
@hongkailiu @jmguzik Could you help to review it, thanks.

**Comment by sgaoshang** [comment]
@jianlinliu @jinyunma pls help to review again, thanks.

File: core-services/prow/02_config/_boskos.yaml [hunk 1]
```diff
@@ -1386,7 +1386,12 @@ resources:
   state: free
   type: vsphere-clusterbot-quota-slice
 - names:
-  - ci-segment-223
+  - devqe-segment-223
+  - devqe-segment-224
+  - devqe-segment-225
+  - devqe-segment-226
+  - devqe-segment-227
+  - devqe-segment-228
```
Comment (jianlinliu) [inline core-services/prow/02_config/_boskos.yaml:1394]: I saw flex is using `devqe-segment-221`, are these all the left that we have?

Comment (sgaoshang) [reply core-services/prow/02_config/_boskos.yaml:1394]: yes, devqe-segment-221 and devqe-segment-222 is for QE/Dev manual testing

**Comment by sgaoshang** [comment]
@smg247 could you help to approve it, thanks.

**Pull request merged**